### PR TITLE
Add joelwmale/codeception-action

### PR DESCRIPTION
**Pull request by joelwmale**
An action that runs [Codeception](https://codeception.com/) tests.

Also available on the marketplace — https://github.com/marketplace/actions/codeception-action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 ## Contents
 
+- [Awesome Actions ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-actions-awesomehttpsgithubcomsindresorhusawesome)
```
Comment (gep13) [inline README.md]: @joelwmale can I ask why this was added?

File: README.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
...
+- [Contents](#contents)
```
Comment (gep13) [inline README.md]: @joelwmale can I ask why this was added?

File: README.md [hunk 3]
```diff
@@ -28,6 +30,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
   - [Frontend Tools](#frontend-tools)
   - [Internet of Things (IOT)](#internet-of-things-iot)
 - [Tutorials](#tutorials)
+- [License](#license)
```
Comment (gep13) [inline README.md]: @joelwmale this one I can understand, so happy to have this added.

File: README.md (hunk 1 above)
Comment (joelwmale) [reply README.md]: I don't actually know how this got added. I think my vscode extension automatically added this. I will revert this change sorry.

File: README.md (hunk 2 above)
Comment (joelwmale) [reply README.md]: Will also revert this.

**Comment by joelwmale** [comment]
I have fixed up the files. Not sure what was going on. Sorry @gep13

File: README.md (hunk 1 above)
Comment (gep13) [reply README.md]: @joelwmale no need to apologise, just wanted to check what was going on 😄

**Comment by gep13** [comment]
@joelwmale your changes have been merged, thanks for your contribution 👍

**Pull request merged**